meshio/ply/_ply.py

```diff
@@ -35,6 +36,25 @@
 numpy_to_ply_dtype = {numpy.dtype(v): k for k, v in ply_to_numpy_dtype.items()}
 
 
+cell_type_to_count = {
+    "line": 2,
+    "triangle": 3,
+    "quad": 4,
+    "polygon5": 5,
+    "polygon6": 6,
+    "triangle6": 6,
+    "quad8": 8,
+}
+
+_cell_type_from_count = {val: key for (key, val) in cell_type_to_count.items()}
+
+
+def cell_type_from_count(count):
```

Comment:
This isn't unique, so we won't be able to convert some meshes forth and back without change.

Reply:
Yeah, is there any difference between a `polygon6` and a `triangle6`?

Reply:
Yeah, is there any difference between a `polygon6` and a `triangle6`?

Reply:
Yes. A `polygon6` is a polygon with 6 corners, a `triangle6` is a "quadratic" triangle (as used in finite elements, for example). It's geometrically the same, but some formats discriminate between the two, so meshio should, too. Do you know what the PLY specification says? A link in the code would be helpful.

Reply:
OK, sounds like we want `polygon6` and not `triangle6`. 

From the overview section of the [link at the top of that script](https://web.archive.org/web/20161221115231/http://www.cs.virginia.edu/~gfx/Courses/2001/Advanced.spring.01/plylib/Ply.txt) it looks like PLY isn't meant to hold that kind of information but I can't find anything explicit either way. Regardless it looks like `polygon6` is far more common so I think I'll just change it to that.